packages/nextjs/src/server/utils.ts

```diff
@@ -194,3 +194,15 @@ export const handleMultiDomainAndProxy = (clerkRequest: ClerkRequest, opts: Auth
     signInUrl,
   };
 };
+
+export const redirectAdapter = (url: string | URL) => {
+  return NextResponse.redirect(url, { headers: { [constants.Headers.ClerkRedirectTo]: 'true' } });
+};
+
+export function assertKey(key: string, onError: () => never): string {
```

Comment:
🙃 / nit: 
The "assert" prefix here made me think this was an assertion function (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions) 

Can we make it work with the errorThrower and an `: asserts key is string` return type? 
If not, I'd probably inline the check as it's fairly simple: you abstract an `if()` check with the cost of some extra indirection. This is a 🙃 so it's up to you if it's worth it or not!

Reply:
No, because you are already passing in a `string` because we are declaring our constants like `process.env.FOO || ''`.